packages/frontend/src/shared/types/theme.d.ts

```diff
@@ -0,0 +1,63 @@
+import { CSSProperties } from 'react';
+
+declare module '@mui/material/styles' {
+  export interface Theme {
+    shape: {
+      borderRadius: {
+        10: CSSProperties['borderRadius'];
+        20: CSSProperties['borderRadius'];
+        35: CSSProperties['borderRadius'];
+        50: CSSProperties['borderRadius'];
+        100: CSSProperties['borderRadius'];
+        circle: CSSProperties['borderRadius'];
+      };
+    };
+  }
+
+  export interface ThemeOptions {
+    shape: {
+      borderRadius: {
+        10: CSSProperties['borderRadius'];
+        20: CSSProperties['borderRadius'];
+        35: CSSProperties['borderRadius'];
+        50: CSSProperties['borderRadius'];
+        100: CSSProperties['borderRadius'];
+        circle: CSSProperties['borderRadius'];
+      };
+    };
+  }
```

Comment:
I think if we’re keeping the shape property here we should probably have a `shape.ts` alongside the `palette.ts` and `typography.ts` in the theme directory. Otherwise, we should probably remove it or update `shape` to be optional here.

To question three, I think this is a pretty good start to the MUI theming pattern for now. I'm thinking once Laura has established the base design system we'll update our theme base to match.